lib/utils/transform.js

```diff
@@ -0,0 +1,40 @@
+const prettyHrtime = require('pretty-hrtime');
+
+const log = require('../utils/logger');
+
+function deleteAddressProperties(address) {
+  /* eslint-disable no-param-reassign */
+  delete address.line1;
+  delete address.line2;
+  delete address.line3;
+  delete address.city;
+  delete address.county;
+  /* eslint-enable no-param-reassign */
+}
+
+function transform(data) {
+  const startTime = process.hrtime();
+
+  data.forEach((item) => {
+    if (!item.address) {
+      return;
+    }
+    const addressLines = [];
+    if (item.address.line1) { addressLines.push(item.address.line1); }
+    if (item.address.line2) { addressLines.push(item.address.line2); }
+    if (item.address.line3) { addressLines.push(item.address.line3); }
+    if (item.address.city) { addressLines.push(item.address.city); }
+    if (item.address.county) { addressLines.push(item.address.county); }
+    // eslint-disable-next-line no-param-reassign
+    item.address.addressLines = addressLines.join(', ');
```

Comment:
I'm not convinced we should be restructuring the data to make it less versatile, i.e. replacing the addressLines array with a string.
Also, won't this set `addressLines` to an empty array for all the non-pharmacy records?

Reply:
Ah good point, it should be an array. I was getting carried away.
And yes, there should be a check to see if `addressLines` already exists.